pkg/engine/events.go

```diff
@@ -206,12 +207,20 @@ func makeStepEventMetadata(op deploy.StepOp, step deploy.Step, debug bool) StepE
 		diffs = differ.Diffs()
 	}
 
+	var richDiff map[string]plugin.DiffKind
+	if richDiffer, hasRichDiff := step.(interface {
+		RichDiff() map[string]plugin.DiffKind
+	}); hasRichDiff {
+		richDiff = richDiffer.RichDiff()
+	}
```

Comment:
fwiw, i think this would be much easier to read as:

```go
	if richDiffer, hasRichDiff := step.(interface { RichDiff() map[string]plugin.DiffKind }); hasRichDiff {
		richDiff = richDiffer.RichDiff()
	}
```

Reply:
specifically, having my go condition wrap like tihs just broke my brain.

Reply:
I would prefer that as well, but `gofmt` does not.